### PR TITLE
[FLINK-25313] Enable flink runtime web-ui

### PR DESCRIPTION
**Pull request by zuston**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -87,6 +87,7 @@ subprojects {
         shadow "org.apache.flink:flink-java:${flinkVersion}"
         shadow "org.apache.flink:flink-streaming-java_${scalaBinaryVersion}:${flinkVersion}"
         shadow "org.apache.flink:flink-streaming-scala_${scalaBinaryVersion}:${flinkVersion}"
+        shadow "org.apache.flink:flink-runtime-web_${scalaBinaryVersion}:${flinkVersion}"
```
Comment (NicoK) [inline build.gradle:92]:
```suggestion
        
        // allows using Flink's web UI when running in the IDE:
        shadow "org.apache.flink:flink-runtime-web_${scalaBinaryVersion}:${flinkVersion}"
```

**Review by NicoK — CHANGES_REQUESTED** [review]
We actually have the same thing in our fork and it has been working well and should be added for the reasons you mentioned.

Two small things before I would merge it:
1. please add the motivation into the commit message
2. I proposed a change to clarify it in the build file as well - let me know what you think

**Comment by zuston** [comment]
Updated @NicoK

**Review by NicoK — APPROVED** [review]

**Comment by zuston** [comment]
Thanks @NicoK

**Pull request merged**